### PR TITLE
Add rule for detection of untranslated jsx attributes

### PR DESCRIPTION
**Pull request by flepretre**
Add a new rule to detect untranslated jsx attributes.

### CODE REVIEW

File: src/rules/no-text-as-attribute.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    docs: {
+      description: 'ensure that no plain text is used in attributes',
+      catefory: 'Possible errors',
+    },
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          attributes: {
+            type: 'array',
```
Comment (D34THWINGS) [inline src/rules/no-text-as-attribute.js]: Maybe we should specify that it's an array of string?
```json
{
  "type": "array",
  "minItems": 1,
  "items": { "type": "string" },
  "uniqueItems": true
}
```

Comment (flepretre) [reply src/rules/no-text-as-attribute.js]: yes you're right 👍

File: src/rules/no-text-as-attribute.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  meta: {
+    docs: {
+      description: 'ensure that no plain text is used in attributes',
+      catefory: 'Possible errors',
```
Comment (j-allison) [inline src/rules/no-text-as-attribute.js]: Oops!

File: src/rules/no-text-as-attribute.js (hunk 1 above)
Comment (flepretre) [reply src/rules/no-text-as-attribute.js]: Done !

**Pull request merged**